ui/desktop/app/templates/scopes/scope/projects/targets/index.hbs

```diff
@@ -167,6 +167,10 @@
                   @route='scopes.scope.projects.targets.target'
                   @model={{B.data.id}}
                   @query={{hash isConnecting=true}}
+                  disabled={{and
+                    (not B.data.address)
+                    (not B.data.host_sources)
+                  }}
```

Comment:
Non-blocking comment: For the sake of time I am happy with this. After giving it some more thought, we could add this logic to the target model then called `B.data.isConnectable()` (or a better name than that 😆). It would make this easier to understand at a glance.

Reply:
oh great idea, I'll clean this up to use this

Reply:
Yup I agree with this, it would make it easier to read since we use this check in quite a few places

Reply:
I totally agree with this approach, nice thinking!!